packages/embed/src/css/includes/_fullsize.scss

```diff
@@ -0,0 +1,6 @@
+@mixin fullsize {
+  width: 100% !important;
+  height: 100% !important;
+  width: 100vw !important;
+  height: 100vh !important;
+}
```

Comment:
That is really good. We'll be able to switch to another solution once find it by updating just a single place in the code. Nice!